### PR TITLE
PoC: ko py

### PR DESCRIPTION
**Pull request by jonjohnsonjr**

### CODE REVIEW

File: cmd/ko/commands.go [hunk 1]
```diff
@@ -295,4 +320,113 @@ func addKubeCommands(topLevel *cobra.Command) {
 	addTagsArg(run, ta)
 
 	topLevel.AddCommand(run)
+
+	// TODO: Write docs
+	py := &cobra.Command{
+		Use:   "py -f FILENAME",
+		Short: "Print the input files with image references resolved to built/pushed image digests.",
+		Long:  `This sub-command finds import path references within the provided files, builds them into Go binaries, containerizes them, publishes them, and prints the resulting yaml.`,
```
Comment (dprotaso) [inline cmd/ko/commands.go]: It's not clear to me after reading this how it's different from `ko resolve`

Comment (dprotaso) [reply cmd/ko/commands.go]: Ah - I see the details here -  https://github.com/google/go-containerregistry/issues/379

File: cmd/ko/commands.go [hunk 2]
```diff
@@ -295,4 +320,113 @@ func addKubeCommands(topLevel *cobra.Command) {
 	addTagsArg(run, ta)
 
 	topLevel.AddCommand(run)
+
+	// TODO: Write docs
+	py := &cobra.Command{
```
Comment (dprotaso) [inline cmd/ko/commands.go]: `py` - I originally thought we were supporting python.

Call it `copy` please lol

Comment (jonjohnsonjr) [reply cmd/ko/commands.go]: But the puns!

Comment (jonjohnsonjr) [reply cmd/ko/commands.go]: What about `ko migrate`?

comigrate (verb): (chiefly in biochemistry) To migrate together

**Pull request closed without merging**